dockerfiles/agent/ubi.Dockerfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+FROM registry.redhat.io/ubi8/ubi-minimal:8.3
```

Comment:
Please add a comment line
# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal

Helping to see new images

Also, please stick to a given version like `FROM registry.access.redhat.com/ubi8-minimal:8.3-230`